central/image/datastore/store/postgres/store_test.go

```diff
@@ -84,3 +93,53 @@ func (s *ImagesStoreSuite) TestStore() {
 	s.False(exists)
 	s.Nil(foundImage)
 }
+
+func (s *ImagesStoreSuite) TestNVDCVSS() {
+	ctx := sac.WithAllAccess(context.Background())
+
+	source := pgtest.GetConnectionString(s.T())
+	config, err := postgres.ParseConfig(source)
+	s.Require().NoError(err)
+	pool, err := postgres.New(ctx, config)
+	s.NoError(err)
+	defer pool.Close()
+
+	Destroy(ctx, pool)
+
+	gormDB := pgtest.OpenGormDB(s.T(), source)
+	defer pgtest.CloseGormDB(s.T(), gormDB)
+	store := CreateTableAndNewStore(ctx, pool, gormDB, false)
+
+	image := fixtures.GetImage()
+	s.NoError(testutils.FullInit(image, testutils.SimpleInitializer(), testutils.JSONFieldsFilter))
+	nvdCvss := &storage.CVSSScore{
+		Source: storage.Source_SOURCE_NVD,
+		CvssScore: &storage.CVSSScore_Cvssv3{
+			Cvssv3: &storage.CVSSV3{
+				Score: 10,
+			},
+		},
+	}
+	for _, component := range image.GetScan().GetComponents() {
+		for _, vuln := range component.GetVulns() {
+			vuln.CvssMetrics = []*storage.CVSSScore{nvdCvss}
+		}
+
+	}
+
+	s.NoError(store.Upsert(ctx, image))
+	foundImage, exists, err := store.Get(ctx, image.GetId())
+	s.NoError(err)
+	s.True(exists)
+	s.NotEmpty(foundImage)
+
+	cvePgStore := cveStore.CreateTableAndNewStore(ctx, pool, gormDB)
```

Comment:
No need to reinitialize store, you can use the `store` initialized above.

Reply:
one is image store and second one is cveStore

Reply:
Oh right, missed that.